PusherServer/PusherOptions.cs

```diff
@@ -39,6 +42,25 @@ public bool Encrypted
         }
 
         /// <summary>
+        /// The host of the HTTP API endpoint excluding the scheme e.g. api.pusherapp.com
+        /// </summary>
+        /// <remarks>
+        /// If the scheme is included when setting the value it will be stripped from the set value.
+        /// e.g. setting with a value of "https://api.pusherapp.com" will result in a value of "api.pusherapp.com".
+        /// </remarks>
+        public string Host
+        {
+            get
+            {
+                return _host;
+            }
+            set
+            {
+                _host = Regex.Replace(value, "^https?://", string.Empty);
```

Comment:
The best I can do without breaking things - `PusherOptions` was never read-only - is instead throw an exception if somebody tries to pass in a scheme https://github.com/pusher/pusher-http-dotnet/commit/0552d5b4ac44e4324e5d11832ecd6e7d425fa6e7